collector/internal/telemetryapi/client.go

```diff
@@ -107,6 +104,15 @@ func (c *Client) Subscribe(ctx context.Context, extensionID string, listenerURI
 	return string(body), nil
 }
 
+func (c *Client) Subscribe(ctx context.Context, extensionID string, listenerURI string) (string, error) {
+	eventTypes := []EventType{
+		Platform,
+		// Function,
+		// Extension,
```

Comment:
Nit: Do we still need these for later?

Reply:
I keep it there so it doesn't have much difference when compared to upstream. (Easier to sync)